src/logic/DataUploader.ts

```diff
@@ -0,0 +1,31 @@
+const MINUTE = 1000 * 60;
+const INTERVAL_DURATION = 1 * MINUTE;
+
+export default class ServerLogic{
```

Comment:
does this need to be a class or can it be a module like `DummyDataGenerator` or `SerialParser`?

Reply:
yes it is better to be a class, because class can hold static data which I used in this design pattern.